common.proto

```diff
@@ -0,0 +1,18 @@
+option optimize_for = LITE_RUNTIME;
+
+package orwell.messages;
+
+
+// Robot is updating its Ultrasound sensor value
+message Ultrasound {
+	required int64 timestamp = 1;
+	required float ultrasound = 2;
+}
+
+// Robot is updating its Battery level value
+message Battery {
+	required int64 timestamp = 1;
+	required int32 voltageMilliVolt = 2;
+	optional float batteryCurrentAmps = 3;
+	optional float motorCurrentAmps = 4;
```

Comment:
I have not changed this, but I do not understand why this is here. We would need a separate Motor message for this.
And then batteryCurrentAmps  could be renamed.
There is also an inconsistency between batteryCurrentAmps and voltageMilliVolt (they should both be either singular or plural and without or without abbreviations). My preference would go to voltageMilliVolts / currentAmperes.
The other solution is to decide that Battery is not the right name, but then it could become a junk-yard (in that case I would suggest the name ValuesWeDoNotKnowWhereToPut).